src/tracer.js

```diff
@@ -18,14 +19,29 @@ export default class Tracer {
     /**
      * Starts and returns a new Span representing a logical unit of work.
      *
-     * @param  {string|object} nameOrFields - if the given argument is a
+     * For example:
+     *
+     *     // Start a new (parentless) root Span:
+     *     var parent = Tracer.startSpan('DoWork');
+     *
+     *     // Start a new (child) Span:
+     *     var child = Tracer.startSpan('Subroutine', {
+     *         reference: Tracer.childOf(parent.context()),
```

Comment:
fwiw, I was upgrading http://github.com/uber-common/opentracing-python-instrumentation and also run into the same annoying problem that we can no longer pass `parent` span. I ended up creating a helper function for now. Even if we allow `childOf()` to accept null reference to parent context (as in https://github.com/opentracing/opentracing-go/pull/96), callers still need to check if `parent` itself is null.

Having said that, allowing `parent=` optional parameter to `startSpan` feels like it's confusing the API. For Go my suggestion is to support `ChildOf(SpanContext)` and `ChildOfSpan(Span)` helpers, both of which allow nil arguments.